recipes/llama-utils/LICENSE.md

```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is in the tarball. You don't need to add it again here.

Reply:
when i delete the license, i also delete the `license_file` entry in the meta.yaml file then the linter complain about the it.
If i have to leave the `license_file` entry , what should i fill it with ?

Reply:
@ocefpaf  please check here